wazimap_ng/profile/serializers/metrics_serializer.py

```diff
@@ -10,22 +10,30 @@ def get_indicator_data(profile_key_metric, geographies):
     indicator_data = IndicatorData.objects.filter(indicator__profilekeymetrics=profile_key_metric, geography__in=geographies)
     return indicator_data
 
+def get_data_for_key_metric_and_geographies(profile_key_metric, geographies):
+    indicator_data = get_indicator_data(profile_key_metric, geographies)
+    if indicator_data.count() > 0:
+        return indicator_data.first().data
+    return None
 
 def absolute_value(profile_key_metric, geography):
-    data = get_indicator_data(profile_key_metric, [geography]).first().data
-    return MetricCalculator.absolute_value(data, profile_key_metric, geography)
+    indicator_data = get_data_for_key_metric_and_geographies(profile_key_metric, [geography])
+    if indicator_data != None:
```

Comment:
we should likely refactor this at some point using the `NullObject` pattern:
https://refactoring.guru/introduce-null-object